index/search.go

```diff
@@ -23,7 +23,7 @@ func (index *Index) checkIfIndexIsReady() bool {
 	if index.IsReady() {
 		return true
 	}
-	timeout := time.After(indexWaitTimeout)
```

Comment:
This change avoids linter detection. Reviewdog only checks current PR diff for linter errors, but this change creates an error elsewhere (makes a constant unused). Previous linter would have complained.